viewing_party/party.py

```diff
@@ -1,23 +1,266 @@
-# ------------- WAVE 1 --------------------
+# ------------- WAVE 1 --------------------from HP
+
+# def create_movie(title, genre, rating):
+#     if title is None or genre is None or rating is None :
+#         return None
+
+#     new_movie = dict(title = title, genre = genre, rating = rating)
+
+#     return new_movie
+
+# def add_to_watched(user_data, movie):
+#     watched_list = user_data["watched"]
+#     watched_list.append(movie)
+
+#     return user_data
+
+# def add_to_watchlist(user_data, movie):
+#     watchlist = user_data["watchlist"]
+#     watchlist.append(movie)
+
+#     return user_data
+
+# def watch_movie(user_data, movie_title):
+#     movie_title_to_move = None
+
+#     for movie in user_data["watchlist"] :
+#         if movie["title"] == movie_title :
+#             movie_title_to_move = movie
+#             break
+#     if movie_title_to_move is not None:
+#         user_data["watched"].append(movie_title_to_move)
+#         user_data["watchlist"].remove(movie_title_to_move)
+
+#     return user_data
+
+
+# ------------- WAVE 1 --------------------from AT
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        return {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+       }
+    return None

```

Comment:
This works, but there is a slightly more appropriate pattern we can follow. When we have a return statement both in and out of a conditional, we are typically looking at a guard clause and the stated purpose of our function. This particular function creates a movie, so that should be the crux of the function rather than be relegated to a guard clause. We can then check the negative for our guard clause instead.

```suggestion
    if not title or not genre or not rating:
        return None
        
    return {
            "title": title,
            "genre": genre,
            "rating": rating
       }

```

This is a pretty common pattern we'll see as we develop!